docs/documentation/test_matrix.md

```diff
@@ -7,8 +7,22 @@ Devices and configurations TrenchBoot has been tested on.
 | Intel Tiger Lake client                                                      |  TPM 2.0   |  PASS  | UEFI firmware |
 | Intel Kaby Lake server                                                       |  TPM 2.0   |  PASS  | UEFI firmware |
 | Intel Skylake sever                                                          |  TPM 2.0   |  PASS  | UEFI firmware |
+| Dell OptiPlex 9010                                                           |  TPM 1.2   |  PASS  | coreboot SeaBIOS firmware |
 | PC Engines APU2 platform series <br>(AMD family 16h models 30h-3fh embedded) |  TPM 2.0   |  PASS  | coreboot firmware |
 | PC Engines APU2 platform series <br>(AMD family 16h models 30h-3fh embedded) |  TPM 1.2   |  PASS  | coreboot firmware |
+| Asus KGPE-D16 <br>(AMD Opteron family 15h models 00h-0fh server)             |  TPM 1.2   |  PASS  | stock BIOS |
 | Asus KGPE-D16 <br>(AMD Opteron family 15h models 00h-0fh server)             |  TPM 2.0   |  FAIL  | coreboot firmware TPM issue |
 | Supermicro M11SDV-8CT <br>(AMD EPYC 3000 Snowy Owl server)                   |  TPM 2.0   |  PASS  | CSM legacy boot |
-| Supermicro M11SDV-8CT <br>(AMD EPYC 3000 Snowy Owl server)                   |  TPM 2.0   |  FAIL  | UEFI boot |
\ No newline at end of file
+| Supermicro M11SDV-8CT <br>(AMD EPYC 3000 Snowy Owl server)                   |  TPM 2.0   |  FAIL  | UEFI boot |
```

Comment:
Do we have bug tracking for this failure?

Reply:
UEFI boot of Xen was not supported and still isn't. That is the most likely reason of FAIL here.

Not sure if I have ever tested the Supermicro platform by booting TrenchBooted Linux.

Reply:
I would appreciate it if you could create an issue since we reported failure. If no one can refer to what failure we have here then maybe we should remove those lines. But that would mean losing potentially useful information. @macpijan @stacktrust @dpsmith any opinions here?

Reply:
See b1c744ea2b347ff114b70cb5656b42b7b62fb863

Reply:
@SergiiDmytruk thank you